--- a/seaborn/algorithms.py
+++ b/seaborn/algorithms.py
@@ -2,7 +2,7 @@
 from __future__ import division
 import numpy as np
 from scipy import stats
-from six.moves import range
+from .external.six.moves import range
 
 
 def bootstrap(*args, **kwargs):
--- a/seaborn/tests/test_algorithms.py
+++ b/seaborn/tests/test_algorithms.py
@@ -1,6 +1,6 @@
 import numpy as np
 from scipy import stats
-from six.moves import range
+from ..external.six.moves import range
 
 import numpy.testing as npt
 from numpy.testing import assert_array_equal
