From: Yaroslav Halchenko <debian@onerussian.com>
Subject: BF: make some non-functioning doctests correct smoke tests

--- a/seaborn/palettes.py
+++ b/seaborn/palettes.py
@@ -61,15 +61,16 @@ def color_palette(name=None, n_colors=6,
 
     Examples
     --------
-    >>> color_palette("muted")
+    >>> p = color_palette("muted")
 
-    >>> color_palette("Blues_d", 10)
+    >>> p = color_palette("Blues_d", 10)
 
-    >>> color_palette("Set1", desat=.7)
+    >>> p = color_palette("Set1", desat=.7)
 
+    >>> import matplotlib.pyplot as plt
     >>> with color_palette("husl", 8):
-    >>>     f, ax = plt.subplots()
-    >>>     ax.plot(x, y)
+    ...     f, ax = plt.subplots()
+    ...     ax.plot(x, y)                  # doctest: +SKIP
 
     See Also
     --------
--- a/seaborn/rcmod.py
+++ b/seaborn/rcmod.py
@@ -148,13 +148,14 @@ def axes_style(style=None, rc=None):
 
     Examples
     --------
-    >>> axes_style("whitegrid")
+    >>> st = axes_style("whitegrid")
 
     >>> set_style("ticks", {"xtick.major.size": 8, "ytick.major.size": 8})
 
+    >>> import matplotlib.pyplot as plt
     >>> with axes_style("white"):
     ...     f, ax = plt.subplots()
-    ...     ax.plot(x, y)
+    ...     ax.plot(x, y)               # doctest: +SKIP
 
     See Also
     --------
@@ -320,13 +321,14 @@ def plotting_context(context=None, rc=No
 
     Examples
     --------
-    >>> plotting_context("poster")
+    >>> c = plotting_context("poster")
 
-    >>> plotting_context("talk", {"lines.linewidth": 2})
+    >>> c = plotting_context("talk", {"lines.linewidth": 2})
 
+    >>> import matplotlib.pyplot as plt
     >>> with plotting_context("paper"):
     ...     f, ax = plt.subplots()
-    ...     ax.plot(x, y)
+    ...     ax.plot(x, y)                 # doctest: +SKIP
 
     See Also
     --------
